packages/create-gatsby/src/plugin-options-form.ts

```diff
@@ -49,51 +54,50 @@ function docsLink(pluginName: string): string {
   )
 }
 
+const isOptionRequired = ([_, option]: [string, Schema]): boolean =>
+  option?.flags?.presence === `required`
+
+const schemaToChoice = ([name, option]: [string, Schema]): IChoice => {
+  const hasDefaultValue =
+    option.flags?.default &&
+    supportedOptionTypes.includes(typeof option.flags?.default)
```

Comment:
I realise this was how it was before, but I do wonder if there's a better way to do this, using the schema rather than checking the type of the default value

Reply:
a side note on that: right now, IIRC, none of the elements that are required have a default value 🤔 . This is a computation that lives there "for nothing" for now I would say :/

Reply:
Good point. Most of them aren't likely to either, because if there's a default it won't need to be required